sklearn/tests/test_discriminant_analysis.py

```diff
@@ -473,6 +568,68 @@ def test_qda_regularization():
     assert_array_equal(y_pred5, y5)
 
 
+def test_qda_ledoitwolf():
+    # test bad solver
+    clf = QuadraticDiscriminantAnalysis(
+            solver="svd",
+            covariance_estimator=LedoitWolf()
+    )
+    assert_raises(NotImplementedError, clf.fit, X, y)
+
+    # test with unkwown solver
+    clf = QuadraticDiscriminantAnalysis(
+            solver="dummy",
+            covariance_estimator=LedoitWolf()
+    )
+    assert_raises(ValueError, clf.fit, X, y)
+
+    clf = QuadraticDiscriminantAnalysis(
+            solver="lsqr",
+            covariance_estimator=LedoitWolf()
+    )
+    with ignore_warnings():
+        clf.fit(X2, y6)
+    y_pred = clf.predict(X2)
+    assert_array_equal(y_pred, y6)
+
+    # Case n_samples_in_a_class < n_features
+    clf = QuadraticDiscriminantAnalysis(
+            solver="lsqr",
+            covariance_estimator=LedoitWolf()
+    )
+    with ignore_warnings():
```

Comment:
what are these warnings which are being ignored?

Reply:
These are actually not useful I removed them.